src/contconic.jl

```diff
@@ -1114,8 +1114,13 @@ function _sdp1test(solver::Function, vecofvars::Bool, sdpcone, config::TestConfi
 
         @test MOI.canget(instance, MOI.VariablePrimal(), X)
         Xv = MOI.get(instance, MOI.VariablePrimal(), X)
+        Xp = [Xv[1] Xv[2] Xv[4]
+              Xv[2] Xv[3] Xv[5]
+              Xv[4] Xv[5] Xv[6]]
+        @test eigmin(Xp) > -max(atol, rtol)
```

Comment:
Why not just `-atol` ?

Reply:
I copypasted from somewhere else I think but `atol` alone is better indeed

Reply:
I have updated it along with the place I copy pasted it from :)